addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -260,7 +260,7 @@ def _post(self, soft=True):
         # OVERRIDE
         posted = super()._post(soft=soft)
 
-        for move in posted.filtered(lambda m: m.l10n_it_send_state == 'to_send' and m.move_type == 'out_invoice'):
+        for move in posted.filtered(lambda m: m.l10n_it_send_state == 'to_send' and m.move_type == 'out_invoice' and m.company_id.country_id == self.env.ref('base.it')):
```

Comment:
country_id.code == 'IT' is faster.  Indeed.

Reply:
On the other hand in send_pec_mail it is doing quite some filtering, but only indeed to show the error...  @bfr-o this seems correct to me.

Reply:
@jco-odoo looks good to me